oslc4j-client-bundle/pom.xml

```diff
@@ -0,0 +1,103 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.eclipse.lyo.clients</groupId>
```

Comment:
is it the right group? I don't recall `clients` being multiple

Reply:
Yes. This was the group id in client.java project https://github.com/eclipse/lyo.client/blob/74edf16d37e7ae8d060bf8d108555ab2072812b8/org.eclipse.lyo.client.java/pom.xml#L4